docs/content/usage/front-matter.mdx

```diff
@@ -26,6 +29,22 @@ description: My page description
 ---
 ```
 
+## Component metadata
+
+To auto-populate the `title` and `description` of a component page using metadata from the [primer/component-metadata](https://github.com/primer/component-metadat) repository, add the `componentId` frontmatter variable:
+
+```markdown
+---
+componentId: avatar
+---
+```
+
+<Note>
+
+`componentId` must match the `id` of a component in this [metadata.json](https://github.com/primer/component-metadata/blob/main/metadata.json) file.
```

Comment:
Is there a reason not to open source https://github.com/primer/component-metadata?

Reply:
No particular reason. I was planning to wait until the repo is a little more mature before we make it public.